decidim-comments/app/frontend/comments/comment.fragment.graphql

```diff
@@ -1,12 +1,12 @@
 fragment Comment on Comment {
   ...CommentData
-  replies {
+  comments {
```

Comment:
As a future opportunity, maybe we could flatten this query and build the tree it afterwards, and/or use [query batching](http://dev.apollodata.com/core/network.html#query-batching)